src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const populationSpans = document.querySelectorAll('.population');
+  const averageSpan = document.querySelector('.average-population');
+  const totalSpan = document.querySelector('.total-population');
+
+  const totalPopulation = [...populationSpans]
+    .map(span => parseInt(span.textContent.replace(/,/g, '')))
+    .filter(Number.isFinite)
+    .reduce((total, population) => total + population, 0);
+
+  const averagePopulation = totalPopulation / populationSpans.length;
+
+  const formatNumberWithCommas = number =>
+    number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+
+  averageSpan.textContent = formatNumberWithCommas(
+    Math.round(averagePopulation)
+  );
+  totalSpan.textContent = formatNumberWithCommas(totalPopulation);
```

Comment:
you can just use toLocaleString('en-US')
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString